lisa_commands/commands.py

```diff
@@ -459,14 +475,57 @@ def reverse_me(query, _slack_event):
     """
     return query[::-1]
 
+def roll_me(query, _slack_event):
+    """
+    rolls the dice. maximum die size: d1000, maximum number of dice: 300
+    :param query: query string
+    :param _slack_event: a dict of slack event information (unused for this function)
+    :return: a string with the result
+    """
+    # lisa can have little a easter egg, as a treat.
+    # video is the smoke weed every day song
+    if query == "a joint" or query == "a blunt":
+        return ":mokeweed::snoop: https://www.youtube.com/watch?v=JcBIzuuIdOA :mokeweed::snoop:"
+
+    if query == "":
+        roll = [1, 6]
+    else:
+        roll = query.split('d', 1)
+    if len(roll) != 2:
```

Comment:
I think it'd be nice if we could default to 1 when the user doesn't supply a number, but it's not a big deal and might not be a use case people other than me really think of.